### ISSUE TITLE
No season folders on export

### ISSUE THREAD

**Issue by Wheemer** [asciidisco/plugin.video.netflix#269]
*I'm submitting a ...*
  - [x] bug report
  - [ ] feature request
  - [ ] support request

## General infomration
When exporting there are no season folders created.

**Comment by hawkeyexp** [comment]
Thats correct - it was the first release of export and i was waiting for asciidisco for upcoming major changes but sine a long time he hasn't commited any changes :-( It is no bug :-) I will change it to feature :-)

**Comment by Wheemer** [comment]
Maybe there's something in the Youtube Library that could help, since that seems to export youtube shows perfectly

**Comment by Wheemer** [comment]
EmbyCon Dev included a work around for this so they do show up properly even without the season folders
